api/v1beta2/foundationdb_env_variables.go

```diff
@@ -21,6 +21,75 @@
+	// EnvNameClusterFile specifies the path to the cluster file / file containing the connection string.
+	EnvNameClusterFile = "FDB_CLUSTER_FILE"
+
+	// EnvNameBinaryDir specifies the path of the FDB binary's directory
+	EnvNameBinaryDir = "BINARY_DIR"
+
+	// EnvNameAdditionalEnvFile if specified for the `foundationdb-kubernetes-sidecar` and `foundationdb-kubernetes-init`
+	// containers, its content will be sourced before any container command runs, and you can override or define there
+	// any other environment variable; this can be used for example to inject environment variables using a shared volume
+	EnvNameAdditionalEnvFile = "ADDITIONAL_ENV_FILE"
+
+	// EnvNameTLSCaFile specifies the path to the certificate authority file for TLS connections
+	EnvNameTLSCaFile = "FDB_TLS_CA_FILE"
+
+	// EnvNameTLSCert specifies the path to the certificate file for TLS connections
+	EnvNameTLSCert = "FDB_TLS_CERTIFICATE_FILE"
+
+	// EnvNameTLSKeyFile specifies the path to the key file for TLS connections
+	EnvNameTLSKeyFile = "FDB_TLS_KEY_FILE"
+
+	// EnvNameTLSVerifyPeers specifies the peer verification rules for incoming TLS connections to the split-image sidecar.
+	// See https://apple.github.io/foundationdb/tls.html#peer-verification for the format
+	EnvNameTLSVerifyPeers = "FDB_TLS_VERIFY_PEERS"
+
+	// EnvNameFDBNetworkSunsetThing specifies whether to ignore the failure to initialize some of the external clients
+	// TODO FDB 7.3 adds a check for loading external client library, which doesn't work with 6.3.
+	//  Consider remove this option once 6.3 is no longer being used.
+	EnvNameFDBNetworkSunsetThing = "FDB_NETWORK_OPTION_IGNORE_EXTERNAL_CLIENT_FAILURES"
+
+	// EnvNameFDBTraceLogGroup sets the 'LogGroup' attribute with the specified value for all events in the trace output files; default value is 'default'
+	EnvNameFDBTraceLogGroup = "FDB_NETWORK_OPTION_TRACE_LOG_GROUP"
+
+	// EnvNameFDBTraceLogDirPath enables trace logs output to a file in the given directory
+	EnvNameFDBTraceLogDirPath = "FDB_NETWORK_OPTION_TRACE_ENABLE"
+
+	// EnvNameFDBExternalClientDir specifies path to search for dynamic libraries and adds them to the list of client
+	// libraries for use by the multi-version client API. Must be set before setting up the network.
+	EnvNameFDBExternalClientDir = "FDB_NETWORK_OPTION_EXTERNAL_CLIENT_DIRECTORY"
+
+	// EnvNameClientThreadsPerVersion specifies the number of client threads to be spawned.  Each cluster will be
+	// serviced by a single client thread. Spawns multiple worker threads for each version of the client that is loaded.
+	// Setting this to a number greater than one implies disable_local_client.
+	EnvNameClientThreadsPerVersion = "FDB_NETWORK_OPTION_CLIENT_THREADS_PER_VERSION"
+
+	// EnvNamePublicIP defines the public IP for the split-image-sidecar or unified FDB kubernetes monitor
 	EnvNamePublicIP = "FDB_PUBLIC_IP"
+
+	// EnvNamePodIP specifies the listen address for the split-image-sidecar or unified FDB kubernetes monitor
```

Comment:
That comment is misleading, nor the kubernetes monitor, nor the sidecar will listen on this address. The value of this env variable will be used to set the listen address of the started fdbserver process.